src/main/resources/com/google/api/codegen/nodejs/package.json.snip

```diff
@@ -25,23 +29,38 @@
       "{@metadata.shortName}",
       "{@metadata.fullName}"
     ],
+    "scripts": {
+      "cover": "nyc --reporter=lcov mocha --require intelli-espower-loader test/*.js && nyc report",
+      "docs": "repo-tools exec -- jsdoc -c .jsdoc.js",
+      "generate-scaffolding": "repo-tools generate all && repo-tools generate lib_samples_readme -l samples/ --config ../.cloud-repo-tools.json",
+      "lint": "repo-tools lint --cmd eslint -- src/ samples/ system-test/ test/",
+      "prettier": "repo-tools exec -- prettier --write src/*.js src/*/*.js samples/*.js samples/*/*.js test/*.js test/*/*.js system-test/*.js system-test/*/*.js",
+      "samples-test": "cd samples/ && npm link ../ && npm test && cd ../",
```

Comment:
Is this always a valid script. I assume this will be added to the package.json when samples are introduced to the repo.

Reply:
It might not be always valid (obviously, if there is no samples, the scripts will not work). I'm still considering different options where this stuff can be put.

Reply:
It would be valid to have repo-tools add the samples-test item if and only if it found a `samples/` directory. (Samples will often be added _post hoc_ so toolkit literally can not know.)